assets/sass/_content-post.scss

```diff
@@ -83,6 +83,62 @@
       border: none;
     }
   }
+
+  /* styles for layouts demonstrated doc */
+  .main-layouts {
```

Comment:
can we move this into another custom file? Loading this on dozens of pages might be excessive.

Reply:
Ah, good point.  I'll work on that.